### PR TITLE
Branch fixes

### PR DESCRIPTION
**Pull request by reaperhulk**
This removes the skip logic for tests when serialization is not available. We do not currently have a backend that doesn't support serialization and there's a good argument that we should require all backends to have basic serialization for non-hardware keys.

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -13,3 +13,4 @@ source =
 exclude_lines =
     @abc.abstractmethod
     @abc.abstractproperty
+    @property
```
Comment (alex) [inline .coveragerc]: Why is this needed?

Comment (reaperhulk) [inline .coveragerc]: https://codecov.io/github/pyca/cryptography/tests/test_interfaces.py#l-50 (I also see it locally during coverage. I vaguely recall "missed branch" being why we excluded the abc decorators as well)

Comment (alex) [inline .coveragerc]: Ugh, those aren't about the `@property` lines, those are about those
methods never being invoked. Can you remove this for now and we can think
about alternative approaches?

On Sat, Jun 27, 2015 at 3:22 PM, Paul Kehrer <notifications@github.com>
wrote:

> In .coveragerc
> <https://github.com/pyca/cryptography/pull/2079#discussion_r33415262>:
>
> > @@ -13,3 +13,4 @@ source =
> >  exclude_lines =
> >      @abc.abstractmethod
> >      @abc.abstractproperty
> > +    @property
>
> https://codecov.io/github/pyca/cryptography/tests/test_interfaces.py#l-50
> (I also see it locally during coverage. I vaguely recall "missed branch"
> being why we excluded the abc decorators as well)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/2079/files#r33415262>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084
